tidb-server/main.go

```diff
@@ -148,7 +147,10 @@ func main() {
 	registerMetrics()
 	configWarning := loadConfig()
 	overrideConfig()
-	validateConfig()
+	if err := cfg.Valid(); err != nil {
+		log.Error("invalid config", zap.Error(err))
```

Comment:
We haven't set up log here, better to `fmt.Println`?

Reply:
After testing, there is a default global logger in log package which will output log to `stdout` just like `fmt.Println`.

Reply:
The error message print an error stack, maybe we can only print the root error msg.